ide/refactoring.api/src/org/netbeans/modules/refactoring/spi/impl/RefactoringPanel.java

```diff
@@ -418,8 +434,11 @@ private JButton[] getButtons() {
     private static final byte LOGICAL = 0;
     private static final byte PHYSICAL = 1;
     private static final byte GRAPHICAL = 2;
-    
+
+    private static final boolean PREVIEW = false;
```

Comment:
i think this should be enabled by default to stay consistent with comparable windows (bookmarks etc) which don't have a button.

There are essentially two options to stay consistent:
 - add a button to all windows which have a preview (bookmarks too etc)
 - or don't add a button and always show the preview

I think having no button is still the easiest option + I am not convinced that there is a need for the button.

Reply:
Made `true` by default.